### DISCUSSION TITLE
How to cancel show read title: "Heads Up! Unlock the full potential of your account – confirm your email to enable the full dojo experience!"

### DISCUSSION THREAD

**Discussion by Rochsen** [codewars/codewars.com#3090] [Q&A] [answered]
"Heads Up! Unlock the full potential of your account – confirm your email to enable the full dojo experience!"
This red message show on my home page or kata practice, how could I cancel this message?
Please make it easy or indicative, thanks!

**Comment by saa000000** [comment]
just access your gmail inbox and  confirm your email and done

**Reply by Rochsen** [reply]
thanks for your reply, But, I don't receive any message in my mail? I register codewars by github, or another register mail also not receive any message, so sad.

**Reply by Rochsen** [reply, marked as answer]
Oh, I found codewars the "Account Settings", which the top of page has an button, it could send comfirm message in my mail, And looking for codewars register mail, so I found and confirm, Thank you for your reply!!!!!

**Comment by qutbiddinabdullayev32-spec** [comment]
Bu ilovani ishlatib kurganim emas endi senab kuraman
